Examples/CocoapodsExample/CocoapodsExample/AppDelegate.m

```diff
@@ -20,13 +20,20 @@ @implementation AppDelegate
 
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    NSLog(@"%@",[[NSNull null] description]);
     [SEGAnalytics debug:YES];
     SEGAnalyticsConfiguration *configuration = [SEGAnalyticsConfiguration configurationWithWriteKey:SEGMENT_WRITE_KEY];
     configuration.trackApplicationLifecycleEvents = YES;
     configuration.trackAttributionData = YES;
     configuration.flushAt = 1;
     [SEGAnalytics setupWithConfiguration:configuration];
-    [[SEGAnalytics sharedAnalytics] track:@"Cocoapods Example Launched"];
+    [[SEGAnalytics sharedAnalytics] track:@"Cocoapods Example Launched" properties:@{
```

Comment:
We should move these "edge cases" to a different file to not pollute the examples.